.github/workflows/tools.yml

```diff
@@ -0,0 +1,38 @@
+name: Build tools docker
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: Ref [a branch or a tag, e.g. release/1.13]
+        required: true
+
+jobs:
+  build_tools_docker_arm64:
+    name: build tools docker
+    runs-on: ["self-hosted", "arm64"]
+    env:
+      HUB: gcr.io/tetrate-istio-arm
```

Comment:
is there a good reason to use an arm specific registry for images? can we reuse non arm ones?

Reply:
for non-arm we're just using official images.